### DISCUSSION TITLE
AJAX -> Fetch-API

### DISCUSSION THREAD

**Discussion by jdmoore7** [bigskysoftware/htmx#1045] [General]
Will the dependency on AJAX ever be swapped out for the JS standard Fetch-API? 

I'm not sure that this is necessary, but I've read that AJAX is being phased out in a number of applications in favor of Fetch-API. I have no idea whether AJAX and Fetch are similarly scoped or if there is functionality in AJAX, which HTMX makes use of, which is not available via Fetch-API.

**Comment by prologic** [comment]
Has any work progress on this?
